client/src/components/FeedbackForm.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable jsx-a11y/label-has-for */
+import React, { useState } from "react";
+import "./feedbackForm.css";
+
+const FeedbackForm = ({ student }) => {
+	console.log("student",student);
+	const [feedbackMessage, setFeedbackMessage] = useState("");
+	const [givenFeedback, setGivenFeedback] = useState([]);
+	const [feedbackList, setFeedbackList] = useState([]);
+
+	const handlerSubmit = (e) => {
+		e.preventDefault();
+		setGivenFeedback(feedbackMessage);
+		setFeedbackList([feedbackMessage, ...feedbackList]);
+		setFeedbackMessage("");
+	};
+
+	const handleChange = (e) => {
+		setFeedbackMessage(e.target.value);
+	};
+	return (
+		<div className="flex-container">
```

Comment:
flex-container is a very generic name, I would give it a more descriptive name so it is easier for other contributors to identify it.